src/agent/configuration_parser/include/configuration_parser.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <toml.hpp>
+
+#include <iostream>
+
+namespace configuration
+{
+    class ConfigurationParser
+    {
+    private:
+        std::string configFileName = "wazuh.conf";
```

Comment:
This needs to be modified once the directory structure is defined.

Reply:
Create a define for `wazuh.conf`.

Reply:
I created a constant hidden in the source file for the configuration parser. Since it was not parameterized and used only once in the default constructor there's no need to store it as a class instance member.